### PR TITLE
Update Start-SPOTenantRename.md

### PR DESCRIPTION
**Pull request by MSFT-GRRojas**
Added not available for GCC, GCC High & DoD.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ ms.reviewer:
 ## SYNOPSIS
 
 > [!IMPORTANT]
-> This feature is currently available to organizations that have no more than 10,000 total SharePoint sites and OneDrive accounts combined.
+> This feature is currently available to organizations that have no more than 10,000 total SharePoint sites and OneDrive accounts combined.  Additionally, tenant rename capability is not available for GCC, GCC High & DoD.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename.md:18]:
```suggestion
> This feature is currently available to organizations that have no more than 10,000 total SharePoint sites and OneDrive accounts combined.  Additionally, tenant renaming capability is not available for GCC, GCC High, and DoD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MSFT-GRRojas, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]
I'm ok after Johan's suggestions are applied

**Comment by yogkumgit** [comment]
@MSFT-GRRojas Gentle reminder. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**